apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -89,76 +102,90 @@ export function InstructorAiGenerateDraftEditor({
+                  </ul>
+                  <div
+                    class="d-flex align-items-center justify-content-end flex-grow-1 border-bottom pr-2"
+                  >
+                    <span data-toggle="modal" data-target="#finalizeModal">
+                      <button
+                        type="button"
+                        class="btn btn-sm btn-primary"
+                        data-toggle="tooltip"
+                        title="Finalize a question to use it on assessments and make manual edits"
+                      >
+                        <i class="fa fa-check" aria-hidden="true"></i>
+                        Finalize question
+                      </button>
+                    </span>
                   </div>
                 </div>
+                <div class="app-preview-content">
+                  ${QuestionAndFilePreview({ resLocals, prompts })}
+                </div>
               </div>
-            </div>
+            </main>
           </div>
-          ${FinalizeModal({ csrfToken: resLocals.__csrf_token })}
-        </div>
-        <div class="card-footer">
-          <a href="#" role="button" data-toggle="modal" data-target="#finalizeModal"
-            >Finalize this question</a
-          >
-          to use it on assessments and make manual edits.
         </div>
-      </div>
-    `,
-  });
+        ${FinalizeModal({ csrfToken: resLocals.__csrf_token })}
+      </body>
+      <script>
+        // TODO: something different on narrow viewports?
+        const chatHistory = document.querySelector('.app-chat-history');
+        chatHistory.scrollTop = chatHistory.scrollHeight;
+      </script>
+    </html>
+  `.toString();
 }
 
 function PromptHistory({
   prompts,
   urlPrefix,
+  showJobLogs,
 }: {
   prompts: AiQuestionGenerationPrompt[];
   urlPrefix: string;
+  showJobLogs: boolean;
 }) {
   return prompts
     .filter((prompt) => prompt.prompt_type !== 'auto_revision')
     .map((prompt) => {
+      // TODO: Once we can upgrade to Bootstrap 5.3, we can use the official
+      // `bg-secondary-subtle` class instead of the custom styles here.
       return html`<div class="d-flex flex-row-reverse">
-          <div class="p-3 mb-2 bg-secondary text-white rounded">${prompt.user_prompt}</div>
+          <div class="p-3 mb-2 rounded" style="background: #e2e3e5; max-width: 90%">
+            ${prompt.user_prompt}
+          </div>
         </div>
         <div class="d-flex flex-row">
-          <div class="p-3 mb-2 bg-dark text-white rounded">
+          <div class="p-3 mb-2 rounded" style="max-width: 90%">
             ${prompt.prompt_type === 'initial'
               ? 'We generated a potential question.'
               : 'We have made changes. Please check the preview and prompt for further revisions.'}
             <div>
               ${run(() => {
-                if (!prompt.job_sequence_id) return '';
+                if (!showJobLogs || !prompt.job_sequence_id) return '';
```

Comment:
For now, I opted to continue showing job logs for global admins (they can be useful to us), but they'll be hidden for normal users.